hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/FairCallQueue.java

```diff
@@ -20,11 +20,7 @@
 
 import java.lang.ref.WeakReference;
 
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.Iterator;
-import java.util.AbstractQueue;
-import java.util.HashMap;
+import java.util.*;
```

Comment:
just a minor code style issue. We typically don't do wildcard imports in Hadoop codebase.
You can configure your IDE to not automatically collapse multiple imports into one wildcard import.

Other than that I am +1